### PR TITLE
Accessibility: Update titles across journeys (DO NOT MERGE YET)

### PR DESCRIPTION
**Pull request by stemount**
Provides a new section title in the flows.

### CODE REVIEW

File: sync/par_data.par_data_partnership_type.partnership.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ configuration:
   partnership_type:
     allowed_values:
       direct: Direct
-      coordinated: Co-ordinator
+      coordinated: Co-ordinated
```
Comment (stemount) [inline sync/par_data.par_data_partnership_type.partnership.yml]: oops, shoehorned this in by accident. Just changing a label.

File: sync/par_flows.par_flow.partnership_application.yml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 langcode: en
 status: true
 dependencies: {  }
 _core:
-  default_config_hash: ZMztu9Hkl0CFqBPTpqY8UjErDhNtzJse9XyLtRfRYW4
+  default_config_hash: 1xdDhK8JCyKPoZCSFoadQ2dnGk3_cMOEz2tVqPA6Wh0
 id: partnership_application
 label: 'Partnership Application Flow'
 default_title: 'New Partnership Application'
+default_section_title: 'New partnership application'
```
Comment (kalpaitch) [inline sync/par_flows.par_flow.partnership_application.yml]: It's probably overridden but probably shouldn't have the two title fragments with the same defaults.

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAdviceForm.php [hunk 3]
```diff
@@ -35,6 +35,13 @@ public function getFormId() {
   ];
 
   /**
+   * {@inheritdoc}
+   */
+  public function titleCallback() {
+    return "Primary Authority Advice | Edit document type and regulatory functions";
```
Comment (kalpaitch) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAdviceForm.php]: Pretty sure this form is used in multiple flows, is this really the right title?

Comment (stemount) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAdviceForm.php]: i'd say so

**Comment by stemount** [comment]
@kalpaitch yeah, it won't show the section title unless you actually have a page title in the first place.

**Pull request merged**